### ISSUE TITLE
Предложение по кумулятивной дельте

### ISSUE THREAD

**Issue by Karona21** [SBProX/SBProX-Dev#77]
Кумулятивная дельта отображается гистограммой,  что не очень удобно при ее анализе. Хорошо бы если в настройках этого индикатора можно было указывать  отображение свечами или хотя бы  линейным графиком. Если такая возможность появиться, готов поменять старую версию на новую

**Comment by Alkor135** [comment]
Поддерживаю предложение!
Хочу лишь добавить, что анализировать кум. дельту удобно при масштабировании для графика отображенного на экране, а не с начала дня и не с начала всего загруженного графика.

**Comment by Karona21** [comment]
Совершенно с Вами согласен.

>Четверг, 16 мая 2019, 21:00 +05:00 от Alkor135 <notifications@github.com>:
>
>Поддерживаю предложение!
>Хочу лишь добавить, что анализировать кум. дельту удобно при масштабировании для графика отображенного на экране, а не с начала дня и не с начала всего загруженного графика.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением,
Юра Белкин
b21vek@mail.ru

**Comment by bizlis** [comment]
Поддерживаю, свечами это отличный вариант

**Comment by vomma** [comment]
Поддерживаю в виде свечей(пример на картинке) показывать дельту. Заранее спасибо.
![Screenshot_1](https://user-images.githubusercontent.com/51166186/58564557-945daf00-8246-11e9-9268-b759e72fe005.jpg)

**Comment by Karona21** [comment]
Нет, я думаю в виде графика, а в таком виде в каком вы показываете лучше отображать открытый интерес.



>Среда, 29 мая 2019, 19:19 +05:00 от vomma <notifications@github.com>:
>
>Поддерживаю в виде свечей(пример на картинке) показывать дельту. Заранее спасибо.
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением,
Юра Белкин
b21vek@mail.ru

**Comment by SBProX** [comment]
Линией мы еще можем сделать, а свечами нет. 
Чтобы рисовать свечи, нужно гистограмму считать из тиков, что не предусмотрено в платформе в принципе. Все что считается, считается из баров. Определить динамику дельты в баре в текущей реализации никак не получится, нужно переделывать всю базу данных.

**Comment by Karona21** [comment]
Добрый день! Линией то же нормально. И еще пожелание по поводу открытого интереса- можно сделать его отображение как в скрине(см. вложение)


>Четверг, 30 мая 2019, 16:12 +05:00 от SBPro X <notifications@github.com>:
>
>Линией мы еще можем сделать, а свечами нет.
>Чтобы рисовать свечи, нужно гистограмму считать из тиков, что не предусмотрено в платформе в принципе. Все что считается, считается из баров. Определить динамику дельты в баре в текущей реализации никак не получится, нужно переделывать всю базу данных.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением,
Юра Белкин
b21vek@mail.ru

**Comment by Alkor135** [comment]
> Линией мы еще можем сделать, а свечами нет.
> Чтобы рисовать свечи, нужно гистограмму считать из тиков, что не предусмотрено в платформе в принципе. Все что считается, считается из баров. Определить динамику дельты в баре в текущей реализации никак не получится, нужно переделывать всю базу данных.

Можно и линией. Просто в том виде в каком кум. дельта представлена сейчас, сложно определять дивергенцию, а с линией будет всё проще.

**Comment by Oleg678** [comment]
Поддерживаю данный тикет, можно объединить с моим #352

**Comment by Oleg678** [comment]
> 
> 
> Линией мы еще можем сделать, а свечами нет.
> Чтобы рисовать свечи, нужно гистограмму считать из тиков, что не предусмотрено в платформе в принципе. Все что считается, считается из баров. Определить динамику дельты в баре в текущей реализации никак не получится, нужно переделывать всю базу данных.

Можете ли вы сделать отрисовку дельты барами ?

**Comment by bizlis** [comment]
Поддерживаю предложение! И хотелось бы еще на кум.делте иметь возможность установки уровней.

**Comment by Vitalyi95** [comment]
Поддерживаю, хотя бы линией, уже было бы неплохо!

**Comment by AndrewAK86** [comment]
Здравствуйте! А тотал дельту, можете сделать в виде свечей с тенями? Если да, то добавьте такую функцию пожалуйста.
